### DISCUSSION TITLE
Torque and HP

### DISCUSSION THREAD

**Discussion by Sb0zZ** [agronick/aa-torque#52] [Q&A]
Hi, First of all thank u for creating this app, Is there a way to visualize the real engine torque on the display, instead of just a calculus? I ready some time ago on a discussion that u were saying that u noticed that the Torque visualized Is Just a calculus between two factor (i don't Remember them  now) so i was wondering if the app could read directly the PID of the Torque from the obd, I was wondering the same thing for the HP,  cuz i think It has the same functioning

**Comment by agronick** [comment]
You can derive Horsepower from other values. See https://github.com/agronick/aa-torque/discussions/28 The problem with the calculation Torque uses by default is it is done using acceleration and weight and it reads 0 if you are going at a steady speed. Using that calculation gives you engine horsepower which is more than what you get at the wheels but it isn't an estimate.

**Reply by Sb0zZ** [reply]
So the car usually use this calcolous for the Torque, because, in BMW that have a performance monitor on board as fabrique, works at the same way?

**Comment by agronick** [comment]
I don't think that was translated correctly. I don't really understand what
you are trying to say.

**Reply by Sb0zZ** [reply]
Okok, Sorry last time my girlfriend helped me tò write the message, Maybe tomorrow in can write again
